Rudder.podspec

```diff
@@ -9,9 +9,11 @@ Pod::Spec.new do |s|
   s.homepage         = "https://github.com/rudderlabs/rudder-sdk-ios"
   s.license          = { :type => "Apache", :file => "LICENSE" }
   s.author           = { "Rudderstack" => "arnab@rudderlabs.com" }
-  s.source           = { :git => "https://github.com/rudderlabs/rudder-sdk-ios.git", :tag => "v1.2.2" }
+  s.source           = { :git => "https://github.com/rudderlabs/rudder-sdk-ios.git", :tag => "v1.3.0" }
```

Comment:
Can you use here from `v1.3.0` to `v#{s.version}`

Reply:
Yes